src/Billing/AutoBilling.php

```diff
@@ -0,0 +1,338 @@
+<?php
+
+/*
+ *  package   OpenEMR
+ *  link      http://www.open-emr.org
+ *  author    Sherwin Gaddis <sherwingaddis@gmail.com>
+ *  copyright Copyright (c )2020.. Sherwin Gaddis <sherwingaddis@gmail.com>
+ *  license   https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
+ */
+
+namespace OpenEMR\Billing;
+
+use Exception;
+
+class AutoBilling
+{
+    private $rid;
+    private $pid;
+    private $userid;
+
+    public function __construct()
+    {
+           $this->rid = $this->getMaxBilling();
```

Comment:
cool work @juggernautsei , took a quick look and this methoid is undefined

Reply:
Nice catch. I removed the method because it was not being used in any of the recent iterations of this class.